src/main/webapp/i18n/de/textExercise.json

```diff
@@ -29,7 +29,17 @@
             "checkPlagiarism": "Auf Plagiate prüfen",
             "exampleSubmissionsRequireExercise": "Um eine Beispielabgabe zu erstellen, musst du zunächst die Erstellung der Textaufgabe abschließen. Du wirst automatisch weitergeleitet.",
             "saveSuccessful": "Dein Antwort wurde erfolgreich gespeichert!",
-            "error": "Ein Fehler ist aufgetreten. Bitte versuche es später noch einmal."
+            "error": "Ein Fehler ist aufgetreten. Bitte versuche es später noch einmal.",
+            "clusterStatistics": {
+                "title": "Cluster Statistik",
+                "clusterId": "Cluster ID",
+                "reusedFeedbackRatio": "Quote wiederverwendeter Feedbacks im Cluster",
+                "reusedFeedbackRatioTooltip": "Anzahl der wiederverwendeten automatischen Feedbacks / Clustergröße",
+                "action": "Aktion",
+                "disable": "Deaktivieren",
+                "enable": "Aktivieren",
+                "noClustersFound": "<b>Keine Textcluster für diese Aufgabe gefunden</b>"
```

Comment:
why would you but the <b> into the translation string? you could also put it round the translated string in the html template.

Reply:
Added all changes in this [commit](https://github.com/ls1intum/Artemis/pull/3910/commits/53cb5c5bd74b39052dcba8d6b93ba49306da0586)